extensions/Gadgets/Gadgets_body.php

```diff
@@ -87,7 +87,10 @@ public static function getPreferences( $user, &$preferences ) {
 			}
 
 			if ( $section !== '' ) {
-				$section = wfMsgExt( "gadget-section-$section", 'parseinline' );
+				// begin wikia change
+				// wfMsg cleanup
+				$section = wfMessage( "gadget-section-$section" )->parse();
```

Comment:
In this case, the name of the message is user defined allowing them to group similar gadgets under appropriate heads on gadgets-definition and Special:Gadgets. if you wanted a list of them, you'd have to look them up on Special:PrefixIndex on the wiki the extension is installed on or something similar. The messages from the extension are prefixed with gadgets-*.